src/assets/scripts/client/navigationLibrary/NavigationLibrary.js

```diff
@@ -45,6 +45,33 @@ export default class NavigationLibrary {
     }
 
     /**
+     *
+     * @property drawSids
+     * @return
+     */
+    get drawSids() {
+        return this._sidCollection.draw;
+    }
+
+    /**
+     *
+     * @property sidCollection
+     * @return
+     */
+    get sidCollection() {
+        return this._sidCollection;
+    }
```

Comment:
You are correct.  That is one of the main reasons I use getters as facades like this.

For your second question, sort of.  if one tried to update a property with `instance.sidCollection = []` it would not be allowed.  however, if one tried to update a property with `instance._sidCollection = []` that would be allowed.

Private vars in JS are not really private, as we've discussed before.  This is just a design pattern that I like to use.  Its sort of like a facade.